### PR TITLE
Docs/documentation update

### PR DESCRIPTION
**Pull request by willu47**

### CODE REVIEW

File: docs/manual/How to contribute to the development of the tool.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+=======================================================
+How to contribute to the development of the tool
+=======================================================
+
+All the community is encouraged to collaborate to the maintenance and development of OSeMOSYS by proposing code modifications, enhancements and extensions. The proposers may want to have the modifications reviewed by experts and included in a bi-annual peer-reviewed publication on the status of development of OSeMOSYS. In such case, they shall submit a related `OSeMOSYS Enhancement Proposal (OEP) <https://github.com/OSeMOSYS/OSeMOSYS/blob/master/.github/ISSUE_TEMPLATE/osemosys-enhancement-proposal.md>`_ on the `OSeMOSYS GitHub Repository <https://github.com/OSeMOSYS/OSeMOSYS>`_ following the instructions below:
+ 
+
+-	Sign in or sign up to GitHub and create a new OEP using the GitHub Issue tracker in the `OSeMOSYS GitHub repository <https://github.com/OSeMOSYS/OSeMOSYS>`_.
+
+-	Fill in the OEP following the the template provided `here <https://github.com/OSeMOSYS/OSeMOSYS/blob/master/.github/ISSUE_TEMPLATE/osemosys-enhancement-proposal.md>`_
```
Comment (willu47) [inline docs/manual/How to contribute to the development of the tool.rst]: Here just give a link directly to the issue tracker - the template then appears.

E.g. `https://github.com/OSeMOSYS/OSeMOSYS/issues/new?assignees=abhishek0208%2C+tniet%2C+willu47&labels=OEP&template=osemosys-enhancement-proposal.md&title=%5BOEP%5D+%3CReplace_with_name_of_proposal%3E`

File: docs/manual/How to contribute to the development of the tool.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+=======================================================
+How to contribute to the development of the tool
+=======================================================
+
+All the community is encouraged to collaborate to the maintenance and development of OSeMOSYS by proposing code modifications, enhancements and extensions. The proposers may want to have the modifications reviewed by experts and included in a bi-annual peer-reviewed publication on the status of development of OSeMOSYS. In such case, they shall submit a related `OSeMOSYS Enhancement Proposal (OEP) <https://github.com/OSeMOSYS/OSeMOSYS/blob/master/.github/ISSUE_TEMPLATE/osemosys-enhancement-proposal.md>`_ on the `OSeMOSYS GitHub Repository <https://github.com/OSeMOSYS/OSeMOSYS>`_ following the instructions below:
```
Comment (willu47) [inline docs/manual/How to contribute to the development of the tool.rst]: Link to the issue tracker `https://github.com/OSeMOSYS/OSeMOSYS/issues/new?assignees=abhishek0208%2C+tniet%2C+willu47&labels=OEP&template=osemosys-enhancement-proposal.md&title=%5BOEP%5D+%3CReplace_with_name_of_proposal%3E`

File: docs/manual/How to contribute to the development of the tool.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
+=======================================================
```
Comment (willu47) [inline docs/manual/How to contribute to the development of the tool.rst:1]: Also remove the old reference to contributing - e.g. see https://osemosys.readthedocs.io/en/latest/manual/Advanced%20functionalities.html#modifications-enhancements-and-extensions-of-the-main-code

**Review by AgnesBelt — APPROVED** [review]
All seems to be fine to me.

**Pull request merged**